codes/classical/matrices/space-time_block.yml

```diff
@@ -0,0 +1,49 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: space-time_block
+physical: waves
+logical: matrices
+
+name: 'Space-time block codes'
+introduced: '\cite{doi:10.1109/49.730453}'
+
+description: 'In a space-time block code \(n\) spatially separated channels transmit symbols in \(T\) time slots. These symbols can be arranged in a \(T\times n\) matrix where the columns correspond to the channels and the rows correspond to the time slots.'
+
+protection: 'Provides protection against errors due to thermal noise and destructive interference arising from traversing an environment with scattering, reflection, and/or refraction.'
```

Comment:
Please include setup and statement of the spacetime coding problem; see Sec. 28.2 in Huffman Kim Sole.